LivlReviews.Api/Controllers/UserController.cs

```diff
@@ -63,17 +115,17 @@ public async Task<IActionResult> Register(RegisterRequest request)
     [Route("login")]
     public async Task<ActionResult<LoginResponse>> Authenticate([FromBody] LoginRequest request)
     {
-        if (!ModelState.IsValid)
-        {
-            return BadRequest(ModelState);
-        }
+        if (String.IsNullOrEmpty(request.Email)) return BadRequest("Email is required");
+        if (String.IsNullOrEmpty(request.Password)) return BadRequest("Password is required");
 
         var managedUser = await _userManager.FindByEmailAsync(request.Email!);
         if (managedUser == null)
         {
             return BadRequest("Bad credentials");
         }
 
+        if (managedUser.EmailConfirmed is false) return BadRequest("Bad credentials");
```

Comment:
Maybe change the message to tell what is the problem

Reply:
I wanted all the login related error to be the same so nobody can guess if an account exist, is waiting invitation or just the password is invaid while trying to log in with random emails.

I once heart it's better for sicurity !

Reply:
Ok I agree for the security, but the poor users will be lost 😧

Reply:
You want to display the english api error message to our french mom users ?

Reply:
Not an english message but it can be great if the front have a way to make a difference between the différent cases